containers/ActivityStream/components/ActivityStream.js

```diff
@@ -23,6 +23,9 @@ class ActivityStream extends React.PureComponent {
     if (props.start !== start || props.end !== end)
       props.fetchActivities()
   }
+  componentWillUnmount() {
+    this.props.wipeApi()
```

Comment:
As discussed should `wipeApi` be in PropTypes? Everything else looks good.

Reply:
Added and fixed some tests